rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_wrapper.cpp

```diff
@@ -24,6 +24,8 @@
 
 #include "rosbag2_storage_default_plugins/sqlite/sqlite_exception.hpp"
 
+#include "../logging.hpp"
```

Comment:
It's preferable not to include headers relatively - is this an usual practice in this project?